standalone-metastore/metastore-common/src/test/java/org/apache/hadoop/hive/common/metrics/metrics2/TestCodahaleReportersConf.java

```diff
@@ -39,6 +40,7 @@
 @org.junit.Ignore("HIVE-23945")
 public class TestCodahaleReportersConf {
 
+  public static final Logger LOGGER = LoggerFactory.getLogger(TestCodahaleReportersConf.class);
```

Comment:
we usually use protected/private for loggers

Reply:
will make it private